hudi-utilities/pom.xml

```diff
@@ -254,7 +254,7 @@
     <dependency>
       <groupId>com.twitter</groupId>
       <artifactId>bijection-avro_${scala.binary.version}</artifactId>
-      <version>0.9.3</version>
+      <version>0.9.7</version>
```

Comment:
We must bump up this version number?

Reply:
Hello, if we use version 0.9.3, when we use recordinjection.convert, it will throw an exception：
```
Exception in thread "main" java.io.FileNotFoundException: \Users\pnarang\workspace\bijection\bijection-avro\target\scala-2.11\scoverage-data\scoverage.measurements.
	at java.io.FileOutputStream.open0(Native Method)
	at java.io.FileOutputStream.open(FileOutputStream.java:270)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:213)
	at java.io.FileWriter.<init>(FileWriter.java:107)
	at scoverage.Invoker$$anonfun$1.apply(Invoker.scala:42)
	at scoverage.Invoker$$anonfun$1.apply(Invoker.scala:42)
	at scala.collection.concurrent.TrieMap.getOrElseUpdate(TrieMap.scala:901)
	at scoverage.Invoker$.invoked(Invoker.scala:42)
	at com.twitter.bijection.avro.GenericAvroCodecs$.toBinary(AvroCodecs.scala:222)
```
I think this is because scalac-scoverage-runtime is used in the source code of version 0.9.3,it  may be a bug in this version.When I upgrade the version to 0.9.7, and it is the same as the version in hudi-flink, it will not throw the above exception

Reply:
s.g.